tests/utils/admincli/vmgroupmgmt.go

```diff
@@ -31,16 +31,14 @@ func CreateVMgroup(ip, name, vmName, dsName string) (string, error) {
 	return ssh.InvokeCommand(ip, admincli.CreateVMgroup+name+" --default-datastore="+dsName+admincli.VMlist+vmName)
 }
 
-// DeleteVMgroup method deletes a vmgroup
-func DeleteVMgroup(ip, name string) (string, error) {
-	log.Printf("Deleting a vmgroup [%s] on esx [%s]\n", name, ip)
-	return ssh.InvokeCommand(ip, admincli.RemoveVMgroup+name)
-}
-
-// DeleteVMgroupAndItsVolumes method deletes a vmgroup and removes its volumes
-func DeleteVMgroupAndItsVolumes(ip, name string) (string, error) {
-	log.Printf("Deleting a vmgroup and its corresponding volumes [%s] on esx [%s]\n", name, ip)
-	return ssh.InvokeCommand(ip, admincli.RemoveVMgroup+name+admincli.RemoveVolumes)
+// DeleteVMgroup method deletes a vmgroup and removes its volumes as well if "delete_vol" is set
+func DeleteVMgroup(ip, name string, delete_vol bool) (string, error) {
+	log.Printf("Deleting a vmgroup [%s] on esx [%s] with delete_vol[%d]\n", name, ip, delete_vol)
+	if delete_vol {
+		return ssh.InvokeCommand(ip, admincli.RemoveVMgroup+name+admincli.RemoveVolumes)
```

Comment:
minor: actually `command` should have been prepared in `if ...else .. ` block that improve readability and `ssh.InvokeCommand(` would be placed at once out of `if` block.